src/Helper/functions.php

```diff
@@ -62,6 +25,7 @@ function mateLoadEnv(string $filename): void
  *
  * Example usage in .mate/services.php:
  * ```php
+ * TODO this is not how the .mate/services.php works. We need to remove this file and use parameters
```

Comment:
hmm but the helper functions would help.zo.make the developer experience better!

why not using the container here? problem is always (globals and also parameters that any extension can enable/disable extensions without the user noticing it. i think we have to solve that and the helper funtions could help with that as we could check the calling stack who is calling that method and disallow all the other calls - exept root config files

what do you think?